x-pack/plugins/monitoring/public/alerts/lib/security_toasts.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiSpacer, EuiLink, EuiCode, EuiText } from '@elastic/eui';
+import { Legacy } from '../../legacy_shims';
+import { toMountPoint } from '../../../../../../src/plugins/kibana_react/public';
+
+export interface AlertingFrameworkHealth {
+  isSufficientlySecure: boolean;
+  hasPermanentEncryptionKey: boolean;
+}
+
+const showTlsAndEncryptionError = () => {
+  const { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } = Legacy.shims.docLinks;
+
+  Legacy.shims.toastNotifications.addWarning({
+    title: toMountPoint(
+      <FormattedMessage
+        id="xpack.monitoring.healthCheck.tlsAndEncryptionErrorTitle"
+        defaultMessage="Additional setup required"
+      />
+    ),
+    text: toMountPoint(
+      <div>
+        <p>
+          {i18n.translate('xpack.monitoring.healthCheck.tlsAndEncryptionError', {
+            defaultMessage: `You must enable Transport Layer Security between Kibana and Elasticsearch 
```

Comment:
I think we probably want to modify this message slightly since the user didn't do any explicit actions to enable alerting and will probably have no context.

Reply:
Good point! What do you think about:
> Alerting features are not working, because your security is currently insufficient. Please enable Transport Layer Security between Kibana and Elasticsearch, and configure an encryption key in your kibana.yml file.

Any suggestions would be helpful

Reply:
I'm not sure honestly. cc @ravikesarwani @lcawl @gchaps for help here